src/main/java/org/hawkular/client/android/backend/model/MetricBucket.java

```diff
@@ -26,6 +26,9 @@
     @SerializedName("end")
     protected long endTimestamp;
 
+    @SerializedName("empty")
+    protected String empty;
```

Comment:
Yeah but while saving it in serialization, that convert to 0/1 and cause pain. So i thought it would be better if it remain String only